packages/flame_test/test/golden_test.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('Golden tests', () {
```

Comment:
Woot! This is awesome to be able to do! I never saw a game engine that had the ability to easily write golden tests as well.